orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -431,8 +454,8 @@ def _get_inference_graph(self, *inputs, **kwargs):
         '''
 
         # Setup dynamic axes for onnx model
-        input_names, dynamic_axes, self._input_names_require_grad = _parse_inputs_for_onnx_export(self._original_module, *inputs, **kwargs)
-        output_names, output_dynamic_axes, self._original_module_output_type = _parse_outputs_for_onnx_export(self._original_module, inputs)
+        input_names, dynamic_axes, self._input_names_require_grad, _ = _parse_inputs_for_onnx_export(self._original_module_input_names, None, *inputs, **kwargs)
```

Comment:
Instead of using `None` or a valid onnx graph as input to decide how to parse input, consider a `ignore_onnx_inference` boolean flag. It declares the intent more clearly and prevent passing class attributes to a method of same class

Reply:
`_parse_inputs_for_onnx_export` is a standalone function, not a class method.